commerce.views.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Implements hook_views_data_alter().
+ */
+function commerce_views_data_alter(array &$data) {
+  // Views field handler to display the entity bundle. If there's only one
+  // bundle for the entity type, don't display anything.
+  $content_entities = array_keys(\Drupal::service('entity_type.repository')->getEntityTypeLabels(TRUE)['Content']);
+  foreach ($content_entities as $content_entity) {
+    $data[$content_entity]['entity_bundle'] = [
```

Comment:
We should be overriding the handler for the existing/real bundle fields instead.